sbi/utils/plot.py

```diff
@@ -277,7 +277,7 @@ def diag_func(row, **kwargs):
                         plt.axvline(
                             single_sample[row],
                             color=opts["samples_colors"][n],
-                            **opts["scatter_diag"]
+                            **opts["scatter_diag"],
```

Comment:
I am not sure these trailing commas are correct in terms of our `black` settings. have you run `black` on all files already?